picard/ui/theme.py

```diff
@@ -141,7 +143,7 @@ def is_dark_theme(self):
 
     @property
     def accent_color(self):  # pylint: disable=no-self-use
-        return None
+        return self._accent_color
```

Comment:
It should still be defined to None if not dark theme.

Reply:
Right, fixed this

Reply:
Did you push the change? ;)

Reply:
Now yes :)